docs/stencil-docs/storefront-customization/theme-accessibility.md

```diff
@@ -0,0 +1,148 @@
+# Developing Themes for Accessibility
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Using alt text](#using-alt-text)
+- [Text accessibility](#text-accessibility)
+- [Keyboard accessibility](#keyboard-accessibility)
+- [Customization](#customization)
+- [Resources](#resources)
+</div>
+
+Designing for accessibility means to be inclusive to the needs of all users. In this tutorial, you will learn how to implement accessibility in your theme and improve theme usability for all users.
```

Comment:
Designing for accessibility means being inclusive to the needs of all users. This tutorial outlines the basic accessibility requirements we would like to see in all BigCommerce themes. We include some examples of each type of change, but it is not an exhaustive list, and we encourage you to do your research to best meet the requirements and help improve usability for all users.